pyrenew/convolve.py

```diff
@@ -165,3 +165,45 @@ def _new_scanner(
         return latest, (new_val, m_net1)
 
     return _new_scanner
+
+
+def compute_delay_ascertained_incidence(
+    p_observed_given_incident: ArrayLike,
+    latent_incidence: ArrayLike,
+    delay_incidence_to_observation_pmf: ArrayLike,
+) -> ArrayLike:
+    """
+    Computes incidences observed according
+    to a given observation rate and based
+    on a delay interval.
+
+    Parameters
+    ----------
+    p_observed_given_incident: ArrayLike
+        The rate at which latent incident counts translate into observed counts.
+        For example, setting ``p_observed_given_incident=0.001``
+        when the incident counts are infections and the observed counts are
+        reported hospital admissions could be used to model disease and population
+        for which the probability (reported) hospital.admission given infection is
+        0.001.
+    latent_incidence: ArrayLike
+        Incidence values based on the true underlying process.
+    delay_incidence_to_observation_pmf: ArrayLike
```

Comment:
Is this description accurate? Conceptually, I think it makes sense that there could be no delay from incidence to reporting. cc @dylanhmorris

Reply:
True. I am in favor of allowing zero delay provided we document this clearly (and also document that we do _not_ allow zero-length generations, so generation interval PMFs are 1-indexed). The only thing this affects one way or the other is how the output vector of predicted observations is indexed in model time.

Reply:
So yes, I favor rewording the docs here accordingly.

Reply:
I am not familiar with the concept of 0/1-indexed pmf. Did I miss anything in the edited docstring below? @dylanhmorris @damonbayer 
```
:math`i^{th}` entry represents a delay of :math:`i` 
time units, i.e. ``delay_incidence_to_observation_pmf[0]`` represents 
the fraction of observations that are not delayed (i.e delayed 0 time unit), 
``delay_incidence_to_observation_pmf[1]`` represents 
the fraction that are delayed 1 time unit, et cetera.
```

Reply:
Yes, that looks good to me! thanks!